main.go

```diff
@@ -37,8 +39,21 @@ func main() {
 }
 
 // Archive returns a b64 encoded tarball of the given migration directory.
-func Archive(path string) (string, error) {
-	dir, err := migrate.NewLocalDir(path)
+func Archive(path string, format atlascloud.DirFormat) (string, error) {
```

Comment:
maybe we can add this in https://github.com/ariga/atlas/blob/7309bafbcd3eabab107ea5db3bed29c244e53c53/sql/migrate/dir.go#L564 and use that?